sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/CardboardParamsUtils.java

```diff
@@ -134,8 +219,8 @@ private static boolean isOriginalCardboardDeviceUri(Uri uri) {
    */
   private static boolean isCardboardDeviceUri(Uri uri) {
     return HTTPS_SCHEME.equals(uri.getScheme())
-        && URI_HOST_GOOGLE.equals(uri.getAuthority())
-        && ("/" + URI_PATH_CARDBOARD_CONFIG).equals(uri.getPath());
+      && URI_HOST_GOOGLE.equals(uri.getAuthority())
```

Comment:
Please avoid making non-related formatting changes like this one on this PR. Please fix here and there.